src/app.ts

```diff
@@ -10,13 +11,24 @@ import { db } from './models';
 
 const port = process.env.PORT || 3000;
 
+const config = {
+  authRequired: false,
+  auth0Logout: true,
+  secret: process.env.SECRET,
```

Comment:
Looks like SECRET is missing from the environment variables in render.com

We should not approve this pull request until environment variable is established.

Reply:
SECRET was added to render environment variables.